packages/search-ui-elasticsearch-connector/src/types.ts

```diff
@@ -0,0 +1,4 @@
+export type ESRequestBody = any;
+export type PostProcessRequestBodyFn = (
+  requestBody: ESRequestBody
+) => ESRequestBody;
```

Comment:
I think we could differentiate Elasticsearch and Enterprise Search in variable names by calling the first `Es` (1 word) and second - `ES` (2 words)
```suggestion
export type EsRequestBody = any;
export type PostProcessRequestBodyFn = (
  requestBody: EsRequestBody
) => EsRequestBody;
```

Reply:
TBH im not super happy with "ES" being in the type name. Was thinking about having a seperate type file for ES and using same names as the official elasticsearch types https://github.com/elastic/elasticsearch-js/blob/main/src/api/types.ts#L937

Reply:
Yeah, good idea.
Perhaps later we could even explore exporting types from that client.